nycrc-package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
can this file be renamed to be a bit clearer? or just published as `@istanbuljs/monorepo-per-package-coverage` or something and then used in the `extends` keys for each of the packages?

Reply:
I'd rather not publish to npm since this is not useful outside our monorepo.  Maybe rename the file to monorepo-per-package-nycrc.json would be better?

Reply:
Why not though? It could be!

Reply:
An advanced example of `nyc merge` would be a good place to start.

Reply:
OK but do you mind if this is done in follow-up?  I don't want to hold up this change while we figure out the documentation, naming of the new package, etc.  I think merge-reports.js would belong in the new package as an exposed bin (nyc-merge-monorepo-reports maybe), probably make it more generic by accepting glob(s) in the argument.